README.md

```diff
@@ -93,32 +50,54 @@ In order to allow the assistant user to create location reservations and service
     - IBM TRIRIGA Workplace Reservation Manager
 
-2.	Make note of the user name and password because it will need to be provided to IBM.
+2.	Make note of the user name and password because it will needed in the provisioning step below.
 3.	Important: Do NOT give the assistant user a primary location.  This user will be used to book rooms and submit service requests on behalf of all users in the TRIRIGA instance.
 
-#### E) (OPTIONAL) TEST THE OSLC ENDPOINTS.
+
+#### C) ADD THE CERTIFICATE AUTHORITY CERTIFICATES TO WEBSERVER JAVA RUNTIME ENVIRONMENT
+
+The provisioning process requires a POST call to the IBM TRIRIGA Assistant services through an Integration Object in TRIRIGA.  In order for this call to be secure, the CA certificates need to be added the JRE used by the WebServer running TRIRIGA.  These instructions are for WebSphere.
+
+1. Make a backup of the `cacerts` file located at `WEBSPHERE_ROOT/AppServer/java/8.0/jre/lib/security`.   
+2. Copy the CA certificates file `triassistant_cacerts` from the `cacerts` directory from this GitHub repository to that same directory.
+3. Execute the `keytool` command to add the CA certificates in `triassistant_cacerts` to the existing `cacerts` file:
+    `../../bin/keytool -import -trustcacerts -alias triassistant -file triassitant-certs -keystore ./cacerts -storepass changeit` 
+4. Restart TRIRIGA
+
+#### D) (OPTIONAL) TEST THE OSLC ENDPOINTS.
 
 If you understand Postman, then you can test the OSLC calls using the Postman collection provided in the postman directory.  You will need to change the payload to have 'location/building/space' you have defined in your TRIRIGA instance. A successful test of the OSLC APIs when there are no OSLC errors.
     
 
-### Part 3 - Request the Assistant Integration ID
+### Part 2 - Submit the Provisioning Request and Check the results
+
+#### E) COMPLETE THE PROVISIONING FORM
+
+The OM package imported contains a form that will execute a process of gathering data for the Assistant and providing that data to IBM.  The form asks for the user name and password for the account created in an earlier step.  This is needed so the IBM's TRIRIGA Assistant services can make an OSLC call back into your TRIRIGA instance and gather building and room names.
 
-#### F) REQUEST YOUR ASSISTANT INTEGRATION ID.
+1. From the TRIRIGA Main Page, click on Requests > Request IBM TRIRIGA Assistant  (if you do not see this menu item, check the trouble shooting section below.)
+2. Enter the URL for your TRIRIGA instance. Note: The instance must be accessible without a VPN and through HTTPS. Include the complete path, starting with `https://`.
+3. Enter the user name and password for the assistant account you created earlier.
+4. Enter your name and email address so we can contact you when the assistant services have been provisioned for your instance.
+5. Click Submit.
+Note: You won't get any direct feedback on the form if everything went well or if there were failures.
```

Comment:
Probably best to phrase this as an optional step to check the logs to verify the submission was sent, rather than saying we don't do x.

EDIT: Now that I'm reading below I'd just add a proceed to step F to validate your submission or just nix.